e2e/helpers.js

```diff
@@ -116,7 +116,7 @@ export function checkIfExists(elementId) {
 export function checkIfElementByTextIsVisible(text) {
   return waitFor(element(by.text(text)))
     .toBeVisible()
-    .withTimeout(25000);
+    .withTimeout(10000);
```

Comment:
why this?

Reply:
Nothing should take more than 10 sec. it's very annoying to have such a high timeout val like 25 while you're debugging, so I changes it to 10